meteor/client/ui/Shelf/AdLibPanel.tsx

```diff
@@ -157,6 +158,16 @@ export function matchFilter(
 		) {
 			return false
 		}
+		// Hide duplicates
```

Comment:
I think this should be explained a bit more. I.e. that when this is enabled, only the very first item, rundown-order-wise, will be displayed, and other items will be hidden. That means that, not only "duplicates in a segment" will be hidden, but also "duplicates in any subsequent segment or rundown" will be hidden. Since this is counter-intuitive to how the rest of Sofie works (i.e. all content relevant to that segment is displayed), this needs to be explained a bit more, both here in the code-comments as well as in the GUI page.

Reply:
I've added this note to the settings 
![image](https://user-images.githubusercontent.com/10697525/113720753-dda2a380-96e6-11eb-93fc-aa9a8d25eb25.png)

As well as expanding that comment, and I've added doc comments to the `matchFilter` function https://github.com/olzzon/tv-automation-server-core/commit/8be6b082af0361d49f938d349d262bbf9536eae9